packages/usa-input-mask/src/index.js

```diff
@@ -0,0 +1,162 @@
+const Sanitizer = require("../../uswds-core/src/js/utils/sanitizer"); // eslint-disable-line no-unused-vars
+
+const MASKED = "masked";
+const MASKED_CLASS = `.${MASKED}`;
+const SHELL = "shell";
+const PLACEHOLDER = "placeholder";
+
+function ready(fn) {
+  document.addEventListener("DOMContentLoaded", fn, false);
+}
+
+ready(() => {
+  const masking = {
+    // User defined Values
+    maskedInputs: document.getElementsByClassName(MASKED),
+    maskedNumber: "XdDmMyY9",
+    maskedLetter: "_",
+
+    init() {
+      masking.setUpMasks(masking.maskedInputs);
+      masking.maskedInputs = document.querySelectorAll(MASKED_CLASS); // Repopulating. Needed b/c static node list was created above.
+      masking.activateMasking(masking.maskedInputs);
+    },
+
+    setUpMasks(inputs) {
+      let i;
+      const l = inputs.length;
+
+      for (i = 0; i < l; i += 1) {
+        masking.createShell(inputs[i]);
+      }
+    },
+
+    // replaces each masked input with a shell containing the input and it's mask.
+    createShell(input) {
+      let text = "";
+      const placeholder = input.getAttribute(`${PLACEHOLDER}`);
+
+      if (
+        placeholder !== null &&
+        placeholder.length > 0 &&
+        placeholder !== " "
+      ) {
+        input.setAttribute("maxlength", placeholder.length);
+        input.setAttribute("data-placeholder", placeholder);
+        input.removeAttribute(`${PLACEHOLDER}`);
+      } else {
+        return;
+      }
+
+      // This section still needs some rework
+      text = `<span class="${SHELL}"><span aria-hidden="true" id="${input.id}Mask"><i></i>${placeholder}</span>${input.outerHTML}</span>`;
+
+      input.outerHTML = text; // eslint-disable-line no-param-reassign, no-unsanitized/property
```

Comment:
We'll need to sanitize if it's taking user input. You can view an example https://github.com/uswds/uswds/blob/cda1459efd1c028fce62c960eafc2653993654f9/packages/usa-date-picker/src/index.js#L871-L877

Reply:
Followed Mitch's lead and used document.createElement(); for this section